Documentation/project-docs/pullrequest-builds.md

```diff
@@ -0,0 +1,92 @@
+- Linux Matrix `eng/pipelines/linux.yml`
+- MacOS Matrix `eng/pipelines/macos.yml`
+
+## Available builds
+
+- corefx-ci (Runs by default on every commit)
+- corefx-outerloop-linux
+- corefx-outerloop-osx
+- corefx-outerloop-windows
+
+### Azure pipelines commands
+
+```
+/azp help (displays help message)
+/azp run (runs all builds listed above)
+/azp run <build name> (/azp run corefx-outerloop-linux)
+```
+
+`/azp` is a shortcut for `/AzurePipelines` which can be used as well.
+
+In order to use comment triggers, at the moment you need to be a `Contributor` or `Member` of the repo.
+
+## How to look at a build failure
+
+First, you need to click in the `details` button on the right hand side of the build you're interested in looking at. That will redirect you to the `Checks` tab within that PR, which will have a summary of that build.
+
+To get to the actual build, you will need an extra step here. You have two options:
+   1. Click on `View more details on Azure Pipelines`:
+   2. Click on `errors / warnings` summary hyperlink.
+
+Once in the build UI you can look at a specific job, or step log, by clicking on them directly.
+
+## How to look at a test failure
+
+Currently, our test results are exposed by https://mc.dot.net/ as we wait for new features by Azure DevOps to use their text explorer. In order to get to the test results, you need to click on the `Send to Helix` step on a job, and within its logs, there will be a text containing the test results URL, which you can `CTRL+Click` to open. It will look like the following:
+```
+Results will be available from https://mc.dot.net/#/user/dotnet-bot/pr~2Fdotnet~2Fcorefx~2Frefs~2Fpull~2F35667~2Fmerge/test~2Ffunctional~2Fcli~2F/20190228.23
+```
+
+Then on Mission Control, you can just navigate through the results and look at stack traces, and detailed logs.
+
+## How to rerun builds
+
+In order to rerun a build, it can be done through comments, the checks tab through the `rerun` button, or in the build UI on the right top corner, under the `...` submenu you will find a `retry` button.
+
+Unfortunately, `rerun` button on checks tab and `retry` button in build's UI are only available for Microsoft people.
+
+## Access a build binlog
+
+We publish binlogs as part of our PR builds to make weird build errors easier to diagnose. In order to find these, you can go to the build UI and find them pretty easily. 
+
+  1. If the build is still running, you can click on the `...` submenu and there will be an `Artifacts` entry.
+  2. If the build is done, on the top right corner, there will be a big blue button called `Artifacts`.
+
+Once you click under the `Artifacts` section, a popup will show, with multiple directories, one for each build leg. Under this directories, you will find every binlog we produce as part of our builds.
+
+# Known Issues
+
+  1. Multiline comments are not supported, so if you want to trigger multiple builds, you have to do it on separate comments.
```

Comment:
I would be a little more clear here, in that a comment trigger meant have _anything_ after it, not even on a new line, as I experienced over here: https://github.com/dotnet/corefx/pull/35322#issuecomment-463836383